users/api/endpoints.go

```diff
@@ -433,7 +433,7 @@ func updateProfilePictureEndpoint(svc users.Service) endpoint.Endpoint {
 			return nil, svcerr.ErrAuthorization
 		}
 
-		user, err := svc.Update(ctx, session, user)
+		user, err := svc.UpdateProfilePicture(ctx, session, user)
```

Comment:
Lets improve the response from this to show the updated profile picture url: 
```json
response:  {
  id: '3cf80c21-a4b5-4c14-b965-12706bcc7cdd',
  first_name: 'wei',
  last_name: 'ying',
  status: 'enabled',
  role: 'user',
  credentials: { username: 'wangxian' },
  email: 'wangxian@example.com',
  created_at: '2024-10-31T19:03:48.588559Z',
  updated_at: '2024-10-31T19:19:20.902556Z',
  updated_by: '3cf80c21-a4b5-4c14-b965-12706bcc7cdd'
}
```

Reply:
> Lets improve the response from this to show the updated profile picture url:
> 
> ```json
> response:  {
>   id: '3cf80c21-a4b5-4c14-b965-12706bcc7cdd',
>   first_name: 'wei',
>   last_name: 'ying',
>   status: 'enabled',
>   role: 'user',
>   credentials: { username: 'wangxian' },
>   email: 'wangxian@example.com',
>   created_at: '2024-10-31T19:03:48.588559Z',
>   updated_at: '2024-10-31T19:19:20.902556Z',
>   updated_by: '3cf80c21-a4b5-4c14-b965-12706bcc7cdd'
> }
> ```

Improve the response to `UpdateRole` as well. If we change the role from `user` to `admin` the response still shows `role: 'user'`